terra-jupyter-base/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.15 - 2023-05-23T18:24:47.726023Z
+
+- Fix cromwell installation, changing the 'cromwell-beta' command to 'cromwell'
```

Comment:
Should this be crom**shell** instead of crom**well**

Reply:
Yes, thanks. I've updated the last commit ([diff](https://github.com/DataBiosphere/terra-docker/compare/ad24f16bd0086d0ca9f029506d940b292bc489e4..86902d34f62d3684ba223ba8d7e9af3f1c2f8c54)).